jobs/presets.yaml

```diff
@@ -1,4 +1,36 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
The status mount needs to move to the bottom section of the presets so it gets mounted on every job not just ones that are image-build:true.

https://github.com/aws/eks-distro-prow-jobs/blob/main/jobs/presets.yaml#L131
https://github.com/aws/eks-distro-prow-jobs/blob/main/jobs/presets.yaml#L125